modules/api/src/main/java/io/fluo/api/config/FluoConfiguration.java

```diff
@@ -192,17 +195,28 @@ public void validate() {
     getWorkerMaxMemory();
     getWorkerNumCores();
     getWorkerThreads();
-    getZookeepers();
     getZookeeperTimeout();
   }
 
-  public FluoConfiguration setZookeepers(String zookeepers) {
+  public FluoConfiguration setApplicationName(String applicationName) {
```

Comment:
should probably check the app name does not have any chars that are not allowed in a zookeeper node name